lib/tmp.ts

```diff
@@ -107,5 +107,10 @@ export function tmpFileDisposer(keepOpen = true): Disposer<TmpFileResult> {
 }
 
 export async function freeSpace() {
-	return (await checkDiskSpace(TMP_DIR)).free;
+	try {
+		return (await checkDiskSpace(TMP_DIR)).free;
+	} catch (error) {
+		console.warn(`Could not check free disk space in "${TMP_DIR}": ${error}`);
+		return 0;
```

Comment:
I'm not sure about the context, but wouldn't it be better to `throw ...` instead of logging and setting the disk space to 0?

Reply:
The point it to avoid the error thrown by checkDiskSpace

Reply:
Forum user reporting this: https://forums.balena.io/t/balenaetcher-portable-versions-1-5-73-not-working-on-haiku-image/131138
I've seen this on sentry too.

Reply:
This way we lose the error message if it's not reported anywhere, that's what I'm worried about, as we wouldn't be able to report it to Sentry either.
Basically it skips decompression this way?

Reply:
It will be in the console.
It happens to very few users and we can't do anything about it.
It will just decompress on the fly instead of decompressing first for these users.